remoteworkitem/trackeritem_repository.go

```diff
@@ -8,3 +8,18 @@ func upload(db *gorm.DB, tqID int, item map[string]string) error {
 	err := db.Create(&ti).Error
 	return err
 }
+
+func convert(db *gorm.DB, tqID int, item map[string]string, provider string) error {
+	ti := TrackerItem{Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
+	remoteTrackerItemMethodRef := RemoteWorkItemImplRegistry[provider]
+	remoteTrackerItem, err := remoteTrackerItemMethodRef(ti)
+	if err != nil {
+		return BadParameterError{parameter: provider, value: nil}
+	}
+	workItem, err := Map(remoteTrackerItem, WorkItemKeyMaps[provider])
+	if err != nil {
+		return ConversionError{simpleError{message: "Error mapping to local work item"}}
+	}
+	err = db.Create(&workItem).Error
```

Comment:
Every time we are creating new work item. This will lots of duplicate entries, right?